Library/Formula/agda.rb

```diff
@@ -17,29 +15,38 @@ class Agda < Formula
     sha256 "d0e17808ee896cdedc499b010b67961dc3db3c430c501faef16c4b4482238447" => :mountain_lion
   end
 
-  head "https://github.com/agda/agda.git", :branch => "master"
+  head do
+    url "https://github.com/agda/agda.git", :branch => "master"
+    resource "stdlib" do
+      url "https://github.com/agda/agda-stdlib.git", :branch => "master"
+    end
+  end
+
+  resource "stdlib" do
+    url "https://github.com/agda/agda-stdlib.git", :branch => "2.4.2.4"
+  end
 
   option "without-stdlib", "Don't install the Agda standard library"
-  option "with-malonzo-ffi",
-    "Include the MAlonzo backend's FFI (depends on the standard library)"
+  option "without-malonzo", "Disable the MAlonzo backend"
 
-  depends_on "ghc" => :build
+  if build.with? "malonzo"
+    depends_on "ghc"
+  else
+    depends_on "ghc" => :build
+  end
```

Comment:
Because `--with-malonzo` is the default, we have CI exercising all the tests.  This option disables a well-defined piece of rarely-used functionality, and doesn’t introduce anything that isn’t being tested.